lib/network/Network.js

```diff
@@ -170,6 +170,21 @@ Network.prototype.setOptions = function (options) {
 
     // the hierarchical system can adapt the edges and the physics to it's own options because not all combinations work with the hierarichical system.
     options = this.layoutEngine.setOptions(options.layout, options);
+    console.log(this.layoutEngine);
+    //
+    // this clears "userControlsFreeAxis" when "hierarchical" is disabled
+    // without doing this, and performing the following steps:
```

Comment:
May still be a good idea to clear the `userControlsFreeAxis` option if `hierarchical` layout is disabled, but the bug I was attempting to fix is probably #4228.